### PR TITLE
[fb] Stop file uploads with restrictive file types for old upload handlers

### PR DESCRIPTION
**Pull request by Harshg999**
## What changes were proposed in this pull request?

- Add check for preventing upload operation for files having restrictive file extensions in old upload handlers also.

## How was this patch tested?

- Manually

### CODE REVIEW

**Comment by Harshg999** [comment]
Skipping unit tests for this change because the code is deprecated and will be removed once new filebrowser is ready. Also E2E testing is more suited for this scenario.

**Review by bjornalm — APPROVED** [review]
LGTM

**Review by agl29 — APPROVED** [review]
LGTM!

File: desktop/core/src/desktop/lib/fs/ozone/upload.py [hunk 1]
```diff
@@ -202,6 +203,12 @@ def __init__(self, request):
 
   def new_file(self, field_name, file_name, *args, **kwargs):
     if self._is_ofs_upload():
+      _, file_type = os.path.splitext(file_name)
+      if RESTRICT_FILE_EXTENSIONS.get() and file_type.lower() in [ext.lower() for ext in RESTRICT_FILE_EXTENSIONS.get()]:
```
Comment (tabraiz12) [inline desktop/core/src/desktop/lib/fs/ozone/upload.py:207]: nit: could be a utility function as this is repeated at 5 places. could be useful later as well.

`if restricted_files():`

**Review by tabraiz12 — APPROVED** [review]

File: desktop/core/src/desktop/lib/fs/ozone/upload.py (hunk 1 above)
Comment (Harshg999) [reply desktop/core/src/desktop/lib/fs/ozone/upload.py:209]: This is handled cleanly in new upload APIs. Old ones are deprecated.

**Pull request merged**